library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Company.CompanyAbstract.orm.xml

```diff
@@ -136,7 +136,7 @@
     </many-to-one>
     <many-to-one field="country" target-entity="Ivoz\Provider\Domain\Model\Country\CountryInterface" fetch="LAZY">
       <join-columns>
-        <join-column name="countryId" referenced-column-name="id" on-delete="CASCADE" nullable="" />
+        <join-column name="countryId" referenced-column-name="id" on-delete="CASCADE" nullable=""/>
```

Comment:
I know there are tons of relations like this, but don't you think it would be a better idea to set a "0" or "false", I mean, for better readability I believe it could be a good option to set imperative that rules

Reply:
I totally agree and I do that with all the new stuff that I add. 
In this case I blame that little change to the helper scripts. Since the line is already modified I don't see any harm in adding the 0.